docs/new-project-setup.md

```diff
@@ -0,0 +1,285 @@
+including using `brew` or `conda`.
+
+Here is a sample for installing the dependencies using `conda`:
+
+```bash
+conda create -n infra python=3.10  # create an environment named 'infra'
+conda activate infra  # activate the environment
+conda install -c conda-forge terraform tflint  # Install terraform and tflint
+```
+
+## Snowflake account setup
+
+### Create the account
+
+1. Assume the `ORGADMIN` role
+1. Under the "Admin" side panel, go to "Accounts" and click the "+ Account" button:
+    1. Select the cloud and region appropriate to the project. The region should be in the United States.
+    1. Select "Business Critical" for the Snowflake Edition.
+    1. You will be prompted to create an initial user with `ACCOUNTADMIN` privileges. This should be you.
+        You will be prompted to create a password for your user. Create one using your password manager,
+        but know that it will ask you change your password upon first log-in.
+    1. Save the Account Locator and URL for your new account.
+1. Log into your new account. You should be prompted to change your password. Save the updated password in your password manager.
+
+### Enable multi-factor authentication for your user
+
+1. Ensure the Duo Mobile app is installed on your phone.
+1. In the upper-left corner, click on your user, and select "Profile"
+1. At the bottom of the dialog, select "Enroll" to enable multi-factor authentication.
+1. Follow the instructions to link the new account with your Duo app.
+
+### Set up key pair authentication
+
+Certain Snowflake clients don't properly cache MFA tokens,
+which means that using them can generate dozens or hundreds of MFA requests on your phone.
+At best this makes the tools unusable, and at worst it can lock your Snowflake account.
+One example of such a tool is (as of this writing) the Snowflake Terraform Provider.
+
+The recommended workaround for this is to add a key pair to your account for use with those tools.
+
+1. Follow the instructions given [here](https://docs.snowflake.com/en/user-guide/key-pair-auth#configuring-key-pair-authentication)
+    to generate a key pair and add the public key to your account.
+    Keep the key pair in a secure place on your device.
+    [This gist](https://gist.github.com/ian-r-rose/1c714ee04be53f7a3fd80322e1a22c27)
+    can be helpful for quickly creating a new encrypted key pair.
+    Usage of the script looks like:
+    ```bash
+    bash generate_encrypted_key.sh <key-name> <passphrase>
+    ```
+1. In your local `.bash_profile` or an `.env` file, add environment variables for
+    `SNOWFLAKE_PRIVATE_KEY_PATH` and (if applicable) `SNOWFLAKE_PRIVATE_KEY_PASSPHRASE`.
+
+### Apply a session policy
+
+By default, Snowflake logs out user sessions after four hours of inactivity.
+ODI's information security policies prefer that we log out after one hour of inactivity for most accounts,
+and after fifteen minutes of inactivity for particularly sensitive accounts.
+
+!!! note
```

Comment:
!!! note is not working in this file or any of our others anymore, not sure why!

Reply:
This uses a markdown extension that is enabled in our docs build, but does not render in GitHub